x-pack/plugins/security_solution/public/common/components/sessions_viewer/index.tsx

```diff
@@ -108,6 +108,7 @@ const SessionsViewComponent: React.FC<SessionsComponentsProps> = ({
         pageFilters={sessionsFilter}
         defaultModel={getSessionsDefaultModel(columns, defaultColumns)}
         end={endDate}
+        bulkActions={false}
```

Comment:
Hi @logeekal, 

I'm just curious, is there any particular reason to not add that feature on the Sessions Tab as well? It seems the only place to be left out.

![image](https://user-images.githubusercontent.com/19270322/196821832-b6406e05-ae56-4d1d-8eb3-f2b197e268bf.png)

Reply:
Hi @opauloh , you were right. I had missed it. But I have now added it to session viewer as well. Thanks for pointing it out.